src/app/workbench/WorkbenchGenericView.tsx

```diff
@@ -194,6 +193,7 @@ export function WorkbenchGenericView({ workbenchIndex, view, chooserOptions }: I
                 role="tabpanel"
                 aria-labelledby="settings-tab"
               >
+                {/* TODO refactor view header such that the logic (using hooks) and the visual representation are separated */}
```

Comment:
Add new issue https://app.zenhub.com/workspaces/reprovisyn-60bf3a424e19290011f57e06/issues/datavisyn/reprovisyn/280 to address this in the future